modules/installation-about-custom-azure-vnet.adoc

```diff
@@ -0,0 +1,68 @@
+
+You need to provide two subnets within your VNet, one for the control plane machines and one for the compute machines. Because Azure distributes machines in different availability zones within the region that you specify, your cluster will have high availability by default.
+
+To ensure that the subnets that you provide to the installation program are suitable, it confirms the following data:
+
+* All the subnets that you specify exist.
+* You provide private subnets.
+* The subnet CIDRs belong to the machine CIDR that you specified.
+* Unless you use a private cluster, you must provide exactly one public and private subnet for each availability zone. Each availability zone contains no more than one public and one private subnet.
+* You provide a public subnet for each private subnet availability zone. Machines will not be provisioned in availability zones that you do not provide private subnets for.
+* If you are not using a private cluster, you provided a public subnet for each availability zone.
+
+If you destroy a cluster that uses an existing VNet, the VNet is not deleted.
+
+[id="installation-about-custom-azure-permissions_{context}"]
+== Division of permissions
+
+Starting with {product-title} 4.3, you do not need all of the permissions that are required for an installation program-provisioned infrastructure cluster to deploy a cluster. This change mimics the division of permissions that you might have at your company: some individuals can create different resource in your clouds than others. For example, you might be able to create application-specific items, like instances, storage, and load balancers, but not networking-related components such as VNets, subnet, or ingress rules.
+
+The Azure credentials that you use when you create your cluster do not need the networking permissions that are required to make VNets and networking core components within the VNet, such as subnets, routing tables, internet gateways, NAT, and VPN. You still need permission to make the application resources that the machines within the cluster require, such as ELBs, security groups, S3 buckets, and nodes.
+
+[id="installation-about-custom-azure-vnet-isolation_{context}"]
+== Isolation between clusters
+
+Because the cluster is unable to modify network security groups in an existing subnet, there is no way to isolate clusters from each other on the VNet.
+////
+These are some of the details from the AWS version, and if any of them are relevant to Azure, they can be included.
+If you deploy {product-title} to an existing network, the isolation of cluster services is reduced in the following ways:
+
+* You can install multiple {product-title} clusters in the same VNet.
+* ICMP ingress is allowed to entire network.
+* TCP 22 ingress (SSH) is allowed to the entire network.
+* Control plane TCP 6443 ingress (Kubernetes API) is allowed to the entire network.
+* Control plane TCP 22623 ingress (MCS) is allowed to the entire network.
+////
```

Comment:
Something else to be aware of, is the NSG's that are created for the subnets need to have rules to allow communication with the control plane, http, https, etc. See  https://github.com/openshift/installer/pull/2521#issuecomment-547190111 for the rules I used.

Reply:
I'm adding a table that contains the ports that we discussed.